app/Http/Controllers/Frontend/RankingController.php

```diff
@@ -198,9 +198,15 @@ private function mode($mode, $hideRanking, $hideRankingGuild)
         }
 
         if ($mode === config('ranking.search-unique')) {
+            /** @var array $uniques */
+            $all_uniques = app('config')->get('unique');
+            foreach ($all_uniques as $key => $unique) {
+                $uniques[] = $key;
+            }
             $jobs = UniqueKillLog::whereNotIn('CharName16', $hideRanking)
```

Comment:
Why are we searching with `whereIn` in that `dbo.uniquekilllogs`? @Ahmed-Elrayes

Reply:
yesterday when i checked the unique ranking after i made the pagination to 15, there was 3 pages and on each page there would be few names from the last page and got values as 0, which means it got the uniques which isn't added in the config/uniques.
so after i made it like this it got the correct values and only made one page.
but today my friend called me and told me that the unique ranking made more than 1 page again and got the same characters but the values are different from the previous page and not 0, so i'm gonna re check the controller to figure out why this happens.
Check those images:

![Image 001](https://user-images.githubusercontent.com/30704271/97164763-4b878e00-178b-11eb-86fd-75a388a9092f.png)

![Image 002](https://user-images.githubusercontent.com/30704271/97164765-4cb8bb00-178b-11eb-80de-3aa16ace2cc7.png)

![Image 003](https://user-images.githubusercontent.com/30704271/97164768-4d515180-178b-11eb-858d-22a749641cb9.png)

![Image 004](https://user-images.githubusercontent.com/30704271/97164771-4de9e800-178b-11eb-8da8-2513844dbf79.png)